SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -628,12 +628,20 @@ void CTurbSSTSolver::BC_Inlet(CGeometry *geometry, CSolver **solver_container, C
 
       conv_numerics->SetPrimitive(V_domain, V_inlet);
 
+      /*--- Non-dimensionalize Inlet_TurbVars if Inlet-Files are used. ---*/
+      su2double Inlet_Vars[MAXNVAR];
+      Inlet_Vars[0] = Inlet_TurbVars[val_marker][iVertex][0];
+      Inlet_Vars[1] = Inlet_TurbVars[val_marker][iVertex][1];
+      if (config->GetInlet_Profile_From_File()) {
+        Inlet_Vars[0] *= 1 / (config->GetVelocity_Ref() * config->GetVelocity_Ref());
+        Inlet_Vars[1] *= config->GetViscosity_Ref() / (config->GetDensity_Ref() * config->GetVelocity_Ref() * config->GetVelocity_Ref());
+      }
+
```

Comment:
Ok now this works but maybe it is not the most performant or elegant solution.

The test I made is to write the example_inlet (by not having an inlet in the first place) and run a simulation with that. Then take the same config without inlet file and run another simulation --> These two simulation should be identical. For SA, the diff between the history files is empty and for SST I see differences in the 9th digit  (the last in that case) popping up after 150 iterations which then accumulate in the residual. I would guess these are floating point errors but the analytical computation is the same.